exchanges/order/order_types.go

```diff
@@ -93,8 +93,12 @@ type Submit struct {
 	// Iceberg specifies whether or not only visible portions of orders are shown in iceberg orders
 	Iceberg bool
 
-	// TradeMode specifies the trading mode for margin and non-margin orders: see okcoin_wrapper.go
-	TradeMode string
+	// IsHFTrade (short for High-Frequency Trade) is used in the KuCoin exchange to determine
+	// whether an order should be executed in high-frequency mode.
+	// For more details, refer to the following documentation:
+	// - https://www.kucoin.com/docs/rest/spot-trading/spot-hf-trade-pro-account/place-hf-order
+	// - https://www.kucoin.com/docs/rest/margin-trading/margin-hf-trade/place-hf-order
+	IsHFTrade bool
```

Comment:
So as suggested in [this comment](https://github.com/thrasher-corp/gocryptotrader/pull/1438#discussion_r1733538627) we can remove `IsHFTrade bool` values for this struct and Cancel struct and completely decouple functionality from SubmitOrder and CancelOrder wrapper functions completely as they cannot be tested satisfactorily. Please add intermediate testing for the individual function calls in the exchange.go file. Access can be provided if required by type assertion.

Reply:
Just meant that for the HF functions that they have testing and coverage.